boltdesignsystem/bolt
# Academy-related SVG Updates

## Jira
http://vjira2:8080/browse/BDS-1950?filter=-1

## Summary
Icon-related updates to support the broader Academy work.

## Details
- Adjusts our existing `icon-generator.js` logic to not mistakenly rename `width-` or `height-` attributes, ONLY `width` and `height` attributes. Ensures these SVG icons are processed correctly given that they use attributes our original build wasn't accounting for. 
- Adds ~10 new SVG icons
- Updates icon-related supporting files (ex. Icon schema)
- Updates our global `.gitignore` to eventually get us to fully stop tracking any file changes to the dynamically generated `index.js` used in `@bolt/components-icons`

## How to test
- Confirm tests pass as expected
- Review new icons on the Icon component's section to ensure the new icons are showing up
